Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
`xz` to be clear. We're aware there's different schools of thought on the whole naming ordeal, but on OS X it has been the case for a fair while, whether using MacPorts, Homebrew or Fink that `<package manager> install xz` gets you `xz` and `lzma`.

There's only a minimal distinction on Linux as well. `apt-get install xz-utils` for example, installs both the `xz-utils` and `lzma`. Both are technically separate packages, but if you install `xz` you get the latter, and both use the same source tarball.